spec/requests/api/v2/transfers/transfers_spec.rb

```diff
@@ -0,0 +1,241 @@
+      end
+
+      it 'returns the Transfer' do
+        expect(json.dig('data', 'id')).to eq(transfer.id.to_s)
+        expect(json.dig('data', 'type')).to eq('between_plates')
+        expect(json.dig('data', 'attributes', 'uuid')).to eq(transfer.uuid)
+        expect(json.dig('data', 'attributes', 'user_uuid')).to eq(transfer.user.uuid)
+        expect(json.dig('data', 'attributes', 'source_uuid')).to eq(transfer.source.uuid)
+        expect(json.dig('data', 'attributes', 'destination_uuid')).to eq(transfer.destination.uuid)
+        expect(json.dig('data', 'attributes', 'transfers')).to eq(transfer.transfers)
+
+        # We don't want to see the TransferTemplate UUID as it's not fetchable.
+        expect(json.dig('data', 'attributes', 'transfer_template_uuid')).not_to be_present
+      end
+    end
+  end
+
+  describe '#patch a Transfer' do
+    let(:resource_model) { create(:transfer_between_plates) }
+    let(:payload) do
+      {
+        'data' => {
+          'id' => resource_model.id,
+          'type' => 'transfers',
+          'attributes' => {
+            'user_uuid' => '111111-2222-3333-4444-555555666666'
+          }
+        }
+      }
+    end
+
+    it 'finds no route for the method' do
+      expect { api_patch "#{base_endpoint}/#{resource_model.id}", payload }.to raise_error(
+        ActionController::RoutingError
+      )
+    end
+  end
+
+  describe '#post a new Transfer' do
+    let(:user) { create(:user) }
+    let(:source) { create(:transfer_plate) }
+    let(:destination) { create(:plate_with_empty_wells) }
+    let(:transfer_template) { create(:transfer_template) } # BetweenPlates
+
+    context 'with a valid payload' do
+      let(:payload) do
+        {
+          'data' => {
+            'type' => 'transfers',
+            'attributes' => {
+              user_uuid: user.uuid,
+              source_uuid: source.uuid,
+              destination_uuid: destination.uuid,
+              transfer_template_uuid: transfer_template.uuid
+            }
+          }
+        }
+      end
+
+      it 'creates a new resource' do
+        expect { api_post base_endpoint, payload }.to change(Transfer::BetweenPlates, :count).by(1)
+      end
+
+      it 'responds with success' do
+        api_post base_endpoint, payload
+
+        expect(response).to have_http_status(:success)
+      end
+
+      it 'responds with the correct attributes' do
+        api_post base_endpoint, payload
+
+        expect(json.dig('data', 'type')).to eq('transfers')
+        expect(json.dig('data', 'attributes', 'uuid')).to be_present
+        expect(json.dig('data', 'attributes', 'user_uuid')).to eq(user.uuid)
+        expect(json.dig('data', 'attributes', 'source_uuid')).to eq(source.uuid)
+        expect(json.dig('data', 'attributes', 'destination_uuid')).to eq(destination.uuid)
+        expect(json.dig('data', 'attributes', 'transfers')).to eq(transfer_template.transfers)
+      end
+    end
+
+    context 'when providing an invalid payload' do
```

Comment:
Could you use your shared examples here `it_behaves_like 'a POST request with a missing/ wrong attribute’` etc just because the below shared example `it must not contain that attribute` is only being used once which seems similar to those other shared examples but I might be wrong! Just doing a quick scan